src/components/selectionBoxLayer.ts

```diff
@@ -150,8 +175,9 @@ export module Components {
         this._xScale.offUpdate(this._adjustBoundsCallback);
       }
       this._xScale = xScale;
+      this._xBoundsMode = PropertyMode.VALUE;
```

Comment:
I needed to put this here to satisfy the case where a user only attaches a scale to a `SelectionBoxLayer`